mlflow/utils/mime_type_utils.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pathlib
+
+from mimetypes import guess_type
+
+# from mlflow.models.model import MLMODEL_FILE_NAME
+# from mlflow.projects._project_spec import MLPROJECT_FILE_NAME
+
+MLMODEL_FILE_NAME = "MLmodel"
+MLPROJECT_FILE_NAME = "mlproject"
```

Comment:
(Just rewriting the constants here is a kludge while I wait for advice on the import issue)